Makefile

```diff
@@ -9,6 +9,7 @@ ifndef NPM
 endif
 
 PHPUNIT="$(PWD)/lib/composer/phpunit/phpunit/phpunit"
+PHPUNITDBG=phpdbg -qrr -d memory_limit=4096M -d zend.enable_gc=0 "$(PWD)/../../lib/composer/bin/phpunit"
```

Comment:
These are not being used, and they are a bit misleading because actually we install phpunit locally. We don't use the one from the core repo.

Both lines can be deleted.

Reply:
fixed